WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -908,6 +908,7 @@ class AbstractPostListViewController: UIViewController,
 
             apost.date_created_gmt = Date()
             apost.status = .publish
+            apost.shouldAttemptAutoUpload = true
             self.uploadPost(apost)
             self.updateFilterWithPostStatus(.publish)
```

Comment:
As discussed with @leandroalonso, we will remove these auto-switching of tabs in a separate PR (#12584).